homeassistant/components/ssdp/websocket_api.py

```diff
@@ -47,7 +50,13 @@ def _async_event_message(message: dict[str, Any]) -> None:
     @callback
     def _async_on_data(info: SsdpServiceInfo, change: SsdpChange) -> None:
         if change is not SsdpChange.BYEBYE:
-            _async_event_message({"add": [asdict(info)]})
+            _async_event_message(
+                {
+                    "add": [
+                        {"name": info.upnp.get(ATTR_UPNP_FRIENDLY_NAME), **asdict(info)}
```

Comment:
`.get()` is intended. While a field is supposed to be present its not always going to be as not every device is compliant